include/ringmesh/basic/types.h

```diff
@@ -38,15 +38,17 @@
 #include <geogram/basic/geometry.h>
 #include <geogram/basic/numeric.h>
 
+#include <cmath>
+
 /*!
  * @file Re-definitions of basic types similar to those of Geogram
  */
 namespace RINGMesh
 {
-    /* If you need interger of 8bits of any other one
+    /* If you need integer of 8bits of any other one
      * it is sufficient to write using GEO::Numeric::uint8 in your file.
      *
-     * Dummy variables were removed, the pollute the namespace and
+     * Dummy variables were removed, they polluted the namespace and
      * it is quite easy to do without them.
```

Comment:
I wonder if these comments are still useful

Reply:
Should I remove them completely then ?

Reply:
In my opinion yes you can